src/Microsoft.DotNet.Interactive/Connection/ProxyKernel.cs

```diff
@@ -16,7 +16,7 @@ public sealed class ProxyKernel : Kernel
     private readonly IKernelCommandAndEventSender _sender;
     private readonly IKernelCommandAndEventReceiver _receiver;
     private ExecutionContext _executionContext;
-    private bool _requiresRequestKernelInfoOnFirstCommand = true;
+    private bool _requiresRequestKernelInfoOnFirstCommand;
```

Comment:
Is this being used for anything?

Reply:
I have removed this in 90b7dcf based on our offline conversation.